arthur_bench/models/models.py

```diff
@@ -163,8 +179,18 @@ class TestCaseOutput(BaseModel):
     """
     score: float
     """
-    Score assigned to output
+    Score assigned to output. This field is decprecated, used score_result instead
+    """
+    score_result: ScoreResult
     """
+    Score information about output. Contains float score and / or category description
+    """
+
+    @root_validator(pre=True)
+    def score_result_backwards_compatible(cls, values):
+        if values.get("score_result") is None:
+            values["score_result"] = ScoreResult(score=values.get("score"))
```

Comment:
should `values.get("score")` be validated as non-null?  or do you think its fine to rely on valid data being put in here in the form of old scores?

Reply:
nvm I see where this is instantiated later on